docs/release-guide.md

```diff
@@ -73,30 +73,59 @@ For details look at `.gitlab-ci.yml`
 
 ## MacOS
 
-Ideally, directly after the tag is created, start with the MacOS release. This has to be done manually, for now. There is a script for this:
+Ideally, directly after the tag is created, start with the MacOS release. As the binaries of x86/arm64 are not compatible with each other, we need to build on two MacOS architectures.This has to be done manually, for now. There is a script for this. Start with the build on x86:
+
+### MacOS x64 build
+
+```bash
+./utils/build-osx.sh  --version v2.0.5-pre4 --appleid "Satoshi Nakamoto (appleid)" --mail "satoshi@gmx.com" specterd package upload
```

Comment:
```suggestion
./utils/build-osx.sh  --version v2.0.5-pre4 specterd package upload
```

Reply:
fixed in [b8a3a7c](https://github.com/cryptoadvance/specter-desktop/pull/2446/commits/b8a3a7ce4ef1e1dfedd24e856d765754f1d5843f)